marklogic-data-hub/src/test/ml-modules/root/test/suites/marklogic-data-hub-spark-connector/writeRecords/ingestionWithNullParams.sjs

```diff
@@ -33,6 +33,7 @@ const envelope = record.document.envelope;
 
   //should add default metadata
   test.assertExists(record.metadata.datahubCreatedOn),
+  String(xdmp.version()).startsWith("9") ? test.assertEqual("flow-developer",record.metadata.datahubCreatedBy):
```

Comment:
Let's change this to asserting that datahubCreatedBy exists and is not "". The tests can be run by different users. We don't care about the identity of the user, we just care that the metadata is set to something.